vcd/datasource_vcd_resource_list.go

```diff
@@ -505,6 +506,32 @@ func vappVmList(d *schema.ResourceData, meta interface{}) (list []string, err er
 	return genericResourceList("vcd_vapp_vm", listMode, nameIdSeparator, []string{org.Org.Name, vdc.Vdc.Name, vappName}, items)
 }
 
+func standaloneVmList(d *schema.ResourceData, meta interface{}) (list []string, err error) {
```

Comment:
We have now three VM lists:
* `vcd_vapp_vm` (with optional parent) : lists all VMs within vApps. If a parent name is provided, it will list only the VMs from such vApp.
* `vcd_vm` : lists all standalone VMs
* `vcd_all_vm`: lists all VMs, standalone and non.